pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -5,43 +5,70 @@ title: Upgrade Kommander
-- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned].
-- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre_provisioned].
+- Review the [Platform Application version updates][release_notes] that are part of this upgrade.
+- For air-gapped environments only: Download the [images bundle][images_bundle], [charts bundle][charts_bundle] and [Catalog Application charts bundle][cat_apps_bundle].  
+- For air-gapped environments only: [Load the Docker images into your Docker registry][load_images]
+- For air-gapped environments only: Download the Kommander application definitions:
+
+```bash
+wget "https://downloads.mesosphere.com/dkp/kommander-applications_v2.2.0.tar.gz"
+```
 
 ## Upgrade Kommander
 
-Before running the following command, ensure that your `dkp` configuration **references the Kommander Management cluster**, otherwise it attempts to run the upgrade on the bootstrap cluster. You can do this by setting the `KUBECONFIG` environment variable [to the appropriate kubeconfig file's location][k8s-access-to-clusters].
+Before running the following command, ensure that your `dkp` configuration **references the Kommander Management cluster**, otherwise it attempts to run the upgrade on the bootstrap cluster. You can do this by setting the `KUBECONFIG` environment variable [to the appropriate kubeconfig file's location][k8s_access_to_clusters].
+
+<p class="message--note"><strong>NOTE:</strong> An alternative to initializing the KUBECONFIG environment variable as stated earlier is to use the <code>–kubeconfig=cluster_name.conf</code> flag. This ensures that Kommander upgrades on the workload cluster.</p>
+
+1.  Use the DKP CLI to upgrade Kommander and all the Platform Applications in the Management Cluster:
 
-<p class="message--note"><strong>NOTE:</strong> An alternative to initializing the KUBECONFIG environment variable as stated earlier is to use the <code>–kubeconfig=cluster_name.conf</code> flag. This ensures that Kommander is upgraded on the workload cluster.</p>
+    For air-gapped:
+
+    ```bash
+    dkp upgrade kommander --charts-bundle bundle.tar.gz
+    ```
 
-1.  Use the DKP CLI to upgrade Kommander and all the Platform Applications in the Management cluster:
+    For non air-gapped:
 
     ```bash
     dkp upgrade kommander
     ```
 
-    <A confirmation message appears once upgrade is complete.>
+    <!-- A confirmation message appears once upgrade is complete. -->
```

Comment:
By editing this line, I unintentionally solved conversation. @vespian to provide sample output